spec/factories/case_types.rb

```diff
@@ -156,6 +156,10 @@
       lgfs_roles
     end
 
+    trait :hearing_subsequent_to_sentence do
```

Comment:
:+1: clearer trait name.

Maybe replace all instances of `:hsts` and then remove that trait?

Reply:
yak-shaving but could do

Reply:
Not a blocker.